FileSystemSimulator/IOSystem/IOSystem.cpp

```diff
@@ -0,0 +1,52 @@
+#include "stdafx.h"
+#include "IOSystem.h"
+
+
+IOSystem::IOSystem(Disk* ldisk)
+{
+	this->ldisk = ldisk;
+}
+
+
+IOSystem::~IOSystem()
+{
+}
+
+int * IOSystem::getBlockLocationOnDisk(int blockNumber)
+{
+	int result[3];
+	int cylinderNumber = -1, trackNumber = -1, sectorNumber = -1;
+
+	cylinderNumber = blockNumber / BLOCKS_IN_CYLINDER;
+	trackNumber = (blockNumber % BLOCKS_IN_CYLINDER) / Track::NUMBER_OF_SECTORS;
```

Comment:
shouldnt here be 
```suggestion
	trackNumber = (blockNumber % BLOCKS_IN_CYLINDER) / Cylinder::NUMBER_OF_TRACKS;
```
?